packages/mask/src/plugins/Trader/SNSAdaptor/trending/CoinMetadataTable.tsx

```diff
@@ -88,19 +95,19 @@ export function CoinMetadataTable(props: CoinMetadataTableProps) {
         {
             chainId: trending.coin.chainId!,
             address: trending.coin.contract_address!,
+            pluginID: trending.coin.pluginID ?? NetworkPluginID.PLUGIN_EVM,
```

Comment:
Change to a required field.

```suggestion
            pluginID: trending.coin.pluginID,
```